README.md

```diff
@@ -1,8 +1,27 @@
-## Generate something from GIR
+# GIR
+
+The `GIR` is used to generate both sys level and Rust-user API level.
+
+## Generate FFI (sys level)
+
+You first need to get the Gir file corresponding to the project you want to bind. Then when you have it, you need to list object you want to generate in a toml file.
```

Comment:
May be need link to https://github.com/gtk-rs/sys/blob/master/conf/gir-gtk.toml as example?